api/app/Http/Validators/UserValidator.php

```diff
@@ -0,0 +1,47 @@
+<?php namespace App\Http\Validators;
+
+use App\Services\Validator;
+
+class UserValidator extends Validator
+{
+    /**
+     * Validation rules.
+     *
+     * @var array
+     */
+    protected $rules = [
+        'email' => 'required|email',
+        'first_name' => 'string',
+        'last_name' => 'string',
+        'phone' => 'string',
+        'mobile' => 'string',
+        'timezone_identifier' => 'timezone',
+        'user_type' => 'string|in:public,admin',
```

Comment:
This validation should come from the statically defined user types set in the user model.